lib/checks/keyboard/focus.js

```diff
@@ -0,0 +1,7 @@
+if (node.getAttribute('onFocus') === 'this.blur()') {
+	return false;
```

Comment:
yes, we are at the moment looking at exact match. 

That will also the same thing, and will trigger in the second test, when the onFocus contains `this.blur`. We can add even more special cases other then the current where this test should fail because of `this.blur` if that is wanted.